Update README.md (#9262)
DataDog/integrations-core: amazon_msk/README.md

### Configuration

1. Edit the `amazon_msk.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Amazon MSK performance data. See the [sample amazon_msk.d/conf.yaml][8] for all available configuration options.

> **Note**: If you reuse the sample file, change the file name from `conf.yaml.example` to `conf.yaml`.

Reviewer: ```suggestion
    **Note**: If you reuse the sample file, change the file name from `conf.yaml.example` to `conf.yaml`.
```

>  Note : Make sure to change the file name. The default file name is conf.yaml.example change it to conf.yaml.

Reviewer: ```suggestion
**Note**: If you reuse the sample file, change the file name from `conf.yaml.example` to `conf.yaml`.
```
